tests/test_render_pass.py

```diff
@@ -0,0 +1,124 @@
+from pyvista import _vtk
+from pyvista.plotting.render_passes import RenderPasses
+
+
+# this ideally would be a fixture, but if it's a fixture renderer collects
+# immediately
+def make_passes():
+    ren = _vtk.vtkRenderer()
+    passes = RenderPasses(ren)
+    return ren, passes
+
+
+def test_render_passes_init():
+    ren, passes = make_passes()
+    assert passes._renderer is ren
+    del ren
+
+    # ensure renderer is collected
+    assert passes._renderer is None
+
+
+def test_blur_pass():
+    ren, passes = make_passes()
+    assert not passes._blur_passes
+    blur_pass = passes.add_blur_pass()
+    assert isinstance(blur_pass, _vtk.vtkGaussianBlurPass)
+    assert len(passes._blur_passes) == 1
+
+    passes.remove_blur_pass()
+    assert not passes._blur_passes
+
+    # double pass should work
+    blur_pass = passes.add_blur_pass()
+    blur_pass = passes.add_blur_pass()
+    assert len(passes._blur_passes) == 2
+
+
+def test_ssaa_pass():
+    ren, passes = make_passes()
+    assert not passes._passes
+    ssaa_pass = passes.enable_ssaa_pass()
+    assert isinstance(ssaa_pass, _vtk.vtkSSAAPass)
+    assert 'vtkSSAAPass' in passes._passes
+
+    # enabling again should just do nothing
+    ssaa_pass = passes.enable_ssaa_pass()
+    assert 'vtkSSAAPass' in passes._passes
+
+    # disabling again should just do nothing
```

Comment:
This comment seems off, because disabling it again does a lot. But I don't know if you'd prefer to edit the comment or remove it altogether. Or perhaps you meant to disable it once first.

Reply:
Meant to do it twice, thanks for catching that. Fixed in 69243a9ff453f7d292ce9329026b0ef8c43fa355.